src/aura/HH_AddressMgr/HH_AddressMgr.cmp

```diff
@@ -0,0 +1,159 @@
+<aura:component >
+    <!-- public attributes -->
+    <aura:attribute name="listAddr" type="Address__c[]" access="public" />
+
+    <!-- private attributes -->
+    <aura:attribute name="addrDefault" type="Address__c" access="private" />
+    <aura:attribute name="addrNew" type="Address__c" access="private" />
+    <aura:attribute name="showChangeAddressPopup" type="Boolean" access="private" />
+    <aura:attribute name="iAddrSelected" type="Integer" default="0" access="private" />
+    <aura:attribute name="isExistingAddrSectionOpen" type="Boolean" default="true" access="private" />
+    
+    <!-- events we handle -->
+    <aura:handler name="init" value="{!this}" action="{!c.doInit}" />
+    <aura:handler name="change" value="{!v.listAddr}" action="{!c.onChangeListAddr}" />
+
+    <!-- events we fire -->
+   	<aura:registerEvent name="AddressChangedEvent" type="c:HH_AddressChangedEvent"/>
+    
+    <!-- markup follows... -->
+    <div class="slds-grid slds-p-around--x-small">
+        <div class="slds-card slds-p-around--none">
+            <div class="slds-card__body slds-m-around--x-small slds-p-vertical--none" >
+                <!-- address -->
+                <aura:if isTrue="{!v.addrDefault != null}">
+                    <div class="slds-text-body--small" >
+                        <ui:outputText value="{!v.addrDefault.MailingStreet__c + 
+                        	(v.addrDefault.MailingStreet2__c != null ? '\r' + v.addrDefault.MailingStreet2__c : '')}" /><br/>
```

Comment:
I almost never see `\r` used for breaking a line.  It is usually `\n` where a character is needed to break the line.

Presumably this works as is, here, but I'm curious as to why you used `\r` instead of `\n`?  Did `\n` not work?